app/serializers/lighthouse/people/va_file_number_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Lighthouse::People::VaFileNumberSerializer < ActiveModel::Serializer
```

Comment:
Why the "Lighthouse" in the module name? What is the connection to Lighthouse?

Reply:
The data source we're using for this PR leverages the lighthouse_bgs gem. When I saw there was already a Lighthouse module, seemed to make sense. Do you have a suggestion/preference?

Reply:
The lighthouse_bgs gem was named thusly because it was a fork of another Ruby bgs gem, forked because the Lighthouse API team wanted to add some capabilities and not conflict with other uses of that gem (which was a dubious choice IMO). 

There's nothing in this class specific to Lighthouse, you're not part of the Lighthouse program, it just seems like arbitrary name cruft that will confuse future developers.

Reply:
Understood, thank you for the explanation. Renamed and placed outside of Lighthouse namespace.